simple_network_sim/mixing_matrix.py

```diff
@@ -0,0 +1,132 @@
+"""A mixing matrix descibes how various groups of a population mix together.
+"""
+
+import csv
+import re
+
+AGE_RE = re.compile(r'\[(\d+),\s*(\d+)\)')
+
+class AgeRange:
+    """A helper class for an age range."""
+    def __init__(self, a, b=None):
+        """Initialiser. If b is None, it is assumed that a is a string to be
+        parsed. Otherwise, the age range is assumed to be [a, b).
+        """
+        if b is None:
+            if isinstance(a, tuple) and len(a) == 2:
+                self._lower = a[0]
+                self._upper = a[1]
+            else:
+                match = AGE_RE.match(a)
+                if match:
+                    self._lower = int(match.group(1))
+                    self._upper = int(match.group(2))
+                elif a[-1] == "+":
+                    self._lower = int(a[:-1])
+                    self._upper = -1  # A marker for no upper limit
+                else:
+                    raise Exception(f'Invalid age range specified: "{a}"')
+        else:
+            self._lower = int(a)
+            self._upper = int(b)
+
+    def contains(self, age):
```

Comment:
Maybe define `__contains__` and use the `in` operator below?

Reply:
Done.